src/selectors/Form/types.ts

```diff
@@ -0,0 +1,25 @@
+import { FormInstance } from '../../instances/Form'
+
+/**
+ * A Selector that provide all {@link FormSelector} methods and adds support
+ * for creating {@apilink DynamicFormInstance | DynamicFormInstances} using
+ * the dynamic syntax.
+ *
+ * See {@link FormSelector} to view all available methods.
+ *
+ * See {@doclink requests-and-responses#dynamic-syntax | Dynamic Instances}
+ * documentation to learn more about the dynamic syntax.
+ *
+ * @example Accessing a DynamicFormInstance using the dynamic syntax.
```

Comment:
There's no `DynamicFormInstance` interface, it's just `FormInstance`:

```suggestion
 * @example Accessing a FormInstance using the dynamic syntax.
```